samples/bluetooth/hci_rpmsg/xx_hci_rpmsg.overlay

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&radio {
+	status = "okay";
+	/* This is a number of antennas that are available on antenna matrix
+	 * designed by Nordic. For more information see README.rst.
+	 */
+	dfe-antenna-num = < 12 >;
```

Comment:
```suggestion
	dfe-antenna-num = <12>;
```

Reply:
The file should not be part of the commit. It is removed.